### PR TITLE
PCM-4142: Removing cache from entitlements request

### PR DESCRIPTION
**Pull request by jtrantin**
@vrathee @gandhikeyur @engineersamuel Please review.

https://projects.engineering.redhat.com/browse/PCM-4142

The problem here was the entitlements response was too big and the store in strataCache failed. That caused the promise never being resolved and that lead to the problems in PCM-4142. I just removed a caching for that request.

### CODE REVIEW

File: dist/redhat_access_pcm_ascension_common.js [hunk 1]
```diff
@@ -4159,23 +4159,51 @@
...
 
 		    function fetchSolr(query) {
-		        if (query.q === undefined || query.q === null || query.q === '') throw 'SOLR Query is mandatory';
+		        if (query.q == null || query.q === '') throw 'SOLR Query is mandatory';
 
 		        var url = udsHostName.clone().setPath('/solr');
 		        url.addQueryParam('wt', 'json');
 		        url.addQueryParam('q', query.q);
-		        if (query.fq !== undefined && query.fq !== '') {
+		        if (query.fq != null && query.fq !== '') {
```
Comment (engineersamuel) [inline dist/redhat_access_pcm_ascension_common.js]: Is `!= null` intentional here?  Strict equals is generally safer, unless you specifically want non-strict equals, which maybe we do?

Comment (jtrantin) [inline dist/redhat_access_pcm_ascension_common.js]: dunno.. I just ran `npm run build`, so this must have been changed in some previous PR.

Comment (engineersamuel) [inline dist/redhat_access_pcm_ascension_common.js]: Strange.  Well it should be fine, but just something to be aware of as sometimes non-strict equals can cause problems.  We just need to make a mental note of it.

Comment (jtrantin) [inline dist/redhat_access_pcm_ascension_common.js]: Yep. Looking at the code this probably originates in uds.js.

**Pull request merged**